src/net/larse/lcms/algorithms/VCT.java

```diff
@@ -47,12 +51,8 @@
  */
 public class VCT {
 
-//  public final static class VCTSolver {
```

Comment:
@gorelick, can you give us some guidance on whether or not you want an internal VCTSolver class?  Yang and I think we heard two different things when you were here - either to follow the format of Landtrendr (with internal LandtrendrSolver class) or like EWMA (without internal class).